src/consumption.cpp

```diff
@@ -11,6 +11,7 @@
 #include <optional>
 #include <string>
 #include <tuple>
+#include <type_id.h>
```

Comment:
```suggestion
#include "type_id.h"
```
angled bracket makes it absolute import, which might cause issues

Reply:
Well, it's already been merged, so that's the subject of a follow-up PR is it not? Not like I can retroactively merge the suggestion, haha

Reply:
well, it's minor so we could ignore till an issue pops up.